kmir/src/kmir/kdist/mir-semantics/ty.k

```diff
@@ -121,6 +121,7 @@ syntax PolyFnSig ::= BinderForFnSig
 // Not needed this way. We could just do PolyFnSig ::= binderForFnSig(value: FnSig, boundVars: BoundVariableKindList).
 
 syntax Ty ::= ty(Int, TyKind) [symbol(ty)]
```

Comment:
Is the ty(Int, TyKind) constructor no longer used?
If so, could you remove it? This would include moving the symbol to the ty(Int) constructor, changing this line (that handles creating the type K AST) in convert.py
https://github.com/runtimeverification/mir-semantics/blob/788088af85870d8807a48bd28fff341fb449239b/kmir/src/kmir/convert_json/convert.py#L446
with
```
return KApply('ty', (KToken(str(n), KSort('Int'))))
```
and removing the second argument from the `ty(...)` constructors in the reference.kmir, keeping only the `Int`.

Reply:
We are not using either constructor of `ty` at the moment in the semantics, so we can skip this for now and handle updating the parser later.

Reply:
As long as we don't use it in the execution semantics, the update is not needed for now. The current convert.ty may be deprecated soon anyway. We can handle it in a separate PR.